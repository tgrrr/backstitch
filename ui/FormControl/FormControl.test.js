import React from 'react'
import renderer from 'react-test-renderer'
import { FormControl } from '.'

it('FormControl: default', () => {
    const component = renderer.create(
        <FormControl>
            <>
                FormControl
            </>
        </FormControl>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
