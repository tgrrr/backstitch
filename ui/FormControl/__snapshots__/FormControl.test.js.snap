// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControl: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Formcontrol "
    data-testid="Formcontrol"
  >
    <div
      className="MuiFormControl-root"
    >
      FormControl
    </div>
  </div>
</div>
`;
